.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+*Before* submitting a pull request, please make sure the following is done...
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests!
+3. If you've changed APIs, describe what needs to be updated in documentation.
+4. Ensure the test suite passes (`go test`).
+5. Make sure your code lints (`go fmt && go vet`).
```

Comment:
These will change over time. Perhaps we should just tell them to run `util/ci-test.sh`?